app/controllers/admin/featured_contents_controller.rb

```diff
@@ -25,16 +25,15 @@ def create
       end
     end
 
-    def show
-      return unless @fc.nil?
```

Comment:
This check is useless as we'll get `RecordNotFound` error if wrong `id` is provided. So I've removed it.